app/allocation/controllers/cancel-move.js

```diff
@@ -0,0 +1,49 @@
+const FormWizardController = require('../../../common/controllers/form-wizard')
+const presenters = require('../../../common/presenters')
+const moveService = require('../../../common/services/move')
+
+class CancelMoveController extends FormWizardController {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setAdditionalLocals)
+  }
+
+  setAdditionalLocals(req, res, next) {
+    const {
+      allocation: {
+        moves: { length: movesCount },
+      },
+    } = res.locals
```

Comment:
^^^ this is cool and all (and it is - it never crossed my mind one could access length in a destructuring), but as we're only grabbing one value, this seems a lot more confusing to read than:

`const movesCount = res.locals.allocation.moves.length`

Reply:
(and it's 4 additional lines that codeclimate will nag us about ;)